module/Order/src/Order/Entity/Order.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Order\Entity;
+
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="orders")
+ */
+
+class Order {
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue
+     **/
+    protected $id;
+
+    /**
+     * @ORM\Column(type="integer")
+     **/
+    protected $order_no;
+
+    /**
+     * @ORM\Column(type="datetime")
+     **/
+    protected $created;
+
+    /**
+     * @ORM\OneToOne(targetEntity="Customer")
```

Comment:
I don't know about doctrine but in JPA (Java), we use @ManyToOne in such case. Please confirm.